README.md

```diff
@@ -1 +1,46 @@
 # Veil
```

Comment:
As a punch line we could do something like:
Veil - Make a version control friendly encrypted .env